frontend/src/common/types/permissions/permissions-group-creation-dto.type.ts

```diff
@@ -0,0 +1,7 @@
+type PermissionsGroupCreationDto = {
+  checkbox: JSX.Element;
```

Comment:
Please, take a look on comments related to [components passed within data values](https://github.com/BinaryStudioAcademy/bsa-2022-guruhub/pull/70#discussion_r944937839)